### ISSUE TITLE
Responsive layout for form

### ISSUE THREAD

**Issue by DriveSoft** [premieroctet/next-admin#139]
How about to implement a new option to make the form layout responsive, you could use grid and add tailwind breakpoints to set number of columns in the grid, example:

```
 edit: {  
    gridColumns: {
       sm: 1, // 640px
       md: 2, // 768px
       lg: 3, // 1024px
       xl: 4, // 1280px
    },
    display: ["field1", "field2", "field_n"]
}
```

**Comment by cregourd** [comment]
We've decided to give more flexibility to the style of the form and its elements. With this feature, you'll be able to add class names to `Form` and to each child. 

To define your own breakpoints, you'll need to modify them in `tailwind.config`

**Comment by cregourd** [comment]
This is available on [v3.2.6](https://www.npmjs.com/package/@premieroctet/next-admin/v/3.2.6) 🚀
You can find an example on demo app
Feel free to give feedback
I close this issue
